src/util/graph.h

```diff
@@ -924,8 +960,14 @@ void grapht<N>::output_dot(std::ostream &out) const
     for_each_successor(i, f);
   };
 
-  const auto to_string = [](const node_indext &i) { return std::to_string(i); };
-  output_dot_generic<node_indext>(out, for_each_node, for_each_succ, to_string);
+  const auto to_string = [&](const node_indext &i) {
```

Comment:
why do we need this `&`?

Reply:
Thanks, I replaced the `&` with `this`, we do need to capture `this`.